lib/shared/widgets/menus/character_context_menu/grind_optimizer.widget.dart

```diff
@@ -76,53 +94,39 @@ class CharacterGrindOptimizerWidget extends StatelessWidget {
                 .map((hash) {
                   final item = items[hash];
                   if (item == null) return null;
-                  final average = currentAverage?.floor() ?? 0;
-                  final diff = item.instanceInfo?.primaryStat?.value?.compareTo(average) ?? 0;
-                  String text = "Average".translate(context);
-                  IconData icon = FontAwesomeIcons.equals;
-                  Color bg = context.theme.onSurfaceLayers.layer3;
-                  Color color = context.theme.onSurfaceLayers.layer1;
+                  final average = currentAverage.toInt();
+                  final diff = (item.instanceInfo?.primaryStat?.value ?? average) - average;
+                  String text = "+" + diff.toString();
+                  Color color = Colors.white;
                   if (diff > 0) {
-                    text = "Above".translate(context);
-                    icon = FontAwesomeIcons.solidSquareCaretUp;
-                    bg = context.theme.successLayers.layer0;
-                    color = context.theme.successLayers.layer3;
+                    color = Colors.greenAccent;
                   }
                   if (diff < 0) {
-                    text = "Below".translate(context);
-                    icon = FontAwesomeIcons.solidSquareCaretDown;
-                    bg = context.theme.errorLayers.layer0;
-                    color = context.theme.errorLayers.layer3;
+                    text = diff.toString();
+                    color = Colors.red;
```

Comment:
```suggestion
                    color = context.theme.errorLayers.layer3;
```

Reply:
I think all of the reds in errorLayers are too dark and too hard to read unless we use a different background. I think we should stay with Colors.red, bold font, and textScaleFactor 1.1 unless you have another suggestion.

![Simulator Screen Shot - iPhone 14 Pro Max - 2023-03-27 at 19 40 46](https://user-images.githubusercontent.com/68356139/228113695-2bbfccde-895c-4f53-8e37-8f378d6a1085.png)